templates/index.md.tmpl

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Is this custom terraform-plugin-docs template required?

Reply:
It's only here to provide that message at the bottom, similar to [archive](https://github.com/hashicorp/terraform-provider-archive/blob/c16f42e9a44863a8ea465d889353e898ba0fab03/templates/index.md.tmpl#L11): `This provider requires no configuration. For information on the resources it provides, see the navigation bar.`

Here's the diff (similar to archive)
![image](https://user-images.githubusercontent.com/8650838/218576173-a1a60d0a-a983-47ea-b352-a97054d72121.png)